app/ui-react/packages/ui/src/Integration/Editor/apiProvider/ApiProviderSelectMethod.tsx

```diff
@@ -16,77 +28,40 @@ export class ApiProviderSelectMethod extends React.Component<
   public render() {
     return (
       <>
-        <Text>{this.props.i18nTitle}</Text>
         <Text>{this.props.i18nDescription}</Text>
         <ListView
```

Comment:
It just struck me, I don't think using a `ListView` and `ListItems` is appropriate for this here.  It's a pretty standard form so it should really be using `FormGroup` etc.

Reply:
I can do that, let me make some changes